packages/node/src/network-configuration.ts

```diff
@@ -2,60 +2,71 @@ import RopstenContracts from "@counterfactual/contracts/networks/3.json";
 import RinkebyContracts from "@counterfactual/contracts/networks/4.json";
 import KovanContracts from "@counterfactual/contracts/networks/42.json";
 import {
-  ContractMigration,
-  NetworkContext,
-  networkContextProps
+  DeployedContractNetworksFileEntry,
+  NAMES_OF_DEPLOYED_CONTRACTS_REQUIRED_FOR_COUNTERFACTUAL,
```

Comment:
The names networkContext and networkContextProps should be kept in sync

Reply:
why is it changed to `EXPECTED_CONTRACT_NAMES...`? What's wrong with `networkContextProps`